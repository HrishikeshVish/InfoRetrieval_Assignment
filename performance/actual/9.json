["https://archive.org/details/MSNBCW_20140509_170000_Ronan_Farrow_Daily#start/3368/end/3403", "https://archive.org/details/CNNW_20190714_170000_Fareed_Zakaria_GPS#start/2359/end/2394", "https://archive.org/details/BBCNEWS_20190629_120000_BBC_News#start/274/end/309", "https://archive.org/details/MSNBCW_20140509_170000_Ronan_Farrow_Daily#start/3368/end/3403", "https://archive.org/details/MSNBCW_20140509_170000_Ronan_Farrow_Daily#start/3368/end/3403", "https://archive.org/details/FOXNEWSW_20160715_000000_The_OReilly_Factor#start/2630/end/2665", "https://archive.org/details/MSNBCW_20140509_170000_Ronan_Farrow_Daily#start/3368/end/3403", "https://archive.org/details/BBCNEWS_20190123_020000_BBC_News#start/1120/end/1155", "https://archive.org/details/MSNBCW_20140509_170000_Ronan_Farrow_Daily#start/3368/end/3403", "https://archive.org/details/MSNBCW_20190315_190000_MSNBC_Live_With_Ali_Velshi#start/2914/end/2949"]