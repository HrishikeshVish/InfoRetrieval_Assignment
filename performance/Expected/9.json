["https://archive.org/details/CNNW_20151130_100000_Early_Start_With_John_Berman_and_Christine_Romans#start/3473/end/3508", "https://archive.org/details/MSNBCW_20190315_190000_MSNBC_Live_With_Ali_Velshi#start/2914/end/2949", "https://archive.org/details/FOXNEWSW_20190922_020000_The_Greg_Gutfeld_Show#start/1691/end/1726", "https://archive.org/details/FOXNEWSW_20160715_000000_The_OReilly_Factor#start/2630/end/2665", "https://archive.org/details/CNNW_20190714_170000_Fareed_Zakaria_GPS#start/2359/end/2394", "https://archive.org/details/MSNBCW_20170710_150000_MSNBC_Live_With_Velshi_and_Ruhle#start/3261/end/3296", "https://archive.org/details/MSNBC_20091218_110000_Morning_Joe#start/1820/end/1855", "https://archive.org/details/BBCNEWS_20190629_120000_BBC_News#start/274/end/309", "https://archive.org/details/BBCNEWS_20190123_020000_BBC_News#start/1120/end/1155", "https://archive.org/details/CNNW_20190922_090000_CNN_Newsroom_Live#start/2189/end/2224"]